.github/workflows/pull-request.yml

```
@@ -9,55 +9,63 @@ jobs:
+    #       TIME_ELAPSED=0
+    #       APPROVED=0
 
-          while [ $TIME_ELAPSED -lt $MAX_WAIT_TIME ]; do
-            echo "Fetching PR reviews from GitHub API..."
-            reviews=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-              "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews")
+    #       while [ $TIME_ELAPSED -lt $MAX_WAIT_TIME ]; do
+    #         echo "Fetching PR reviews from GitHub API..."
+    #         reviews=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+    #           "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews")
             
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)